NewBank/newbank/server/NewBank.java

```diff
@@ -52,4 +61,24 @@ private String showMyAccounts(CustomerID customer) {
 		return (customers.get(customer.getKey())).accountsToString();
 	}
 
+	private String addMainAccount(CustomerID customer) {
+		Customer userName = customers.get(customer.getKey());
+		Account main = new Account("Main", 0.00);
+		userName.addAccount(main);
+		return ("SUCCESS New account " + main.toString());
+	}
+
+	private String addSavingsAccount(CustomerID customer) {
+		Customer userName = customers.get(customer.getKey());
+		Account savings = new Account("Savings", 0.00);
+		userName.addAccount(savings);
+		return ("SUCCESS New account " + savings.toString());
+	}
+
+	private String addCheckingAccount(CustomerID customer) {
+		Customer userName = customers.get(customer.getKey());
+		Account checking = new Account("Checking", 0.00);
+		userName.addAccount(checking);
+		return ("SUCCESS New account " + checking.toString());
+	}
```

Comment:
These three methods have a lot of duplicated code - they could be combined into a single method, with a parameter that is the type of account.